code/game/verbs/suicide.dm

```diff
@@ -45,6 +45,8 @@
 		suiciding = TRUE
 		do_suicide()
 		create_log(ATTACK_LOG, "Attempted suicide")
+		if(SSticker.mode.tdm_gamemode)
```

Comment:
Might want to add a check for the RR as well if you're watching for suicides, since it's probably the next closest thing.